software_installation.md

```diff
@@ -53,3 +58,49 @@ We will create a MiniConda environment with the following Python packages:
    It should look like this: ![](./images/choose_kernel.png)
 
 5. Test the environment and make sure that it works as expected.
+
+
+### Environment creation: R
+
+Unlike with Python, it is not possible to add a Jupyter installation in your R conda environment. 
```

Comment:
I was wondering why I didn't come across this issue when I was trying out the IRkernel myself. Now I see that I did that by installing the IRkernel package via R, rather than via Conda, so like this:

```
conda create -n r41env r-base=4.1 r-dplyr r-data.table r-ggplot2 r-tidyr jupyter_client -c conda-forge
conda activate r41env
Rscript -e 'install.packages("IRkernel", repos="https://ftp.belnet.be/mirror/CRAN/")'
Rscript -e 'IRkernel::installspec(prefix="${VSC_HOME}/.local/", name="r41env", displayname="r41env")'
```

Though this is not ideal either (package compiled from source and managed outside Conda), I think it is still be preferable to set things up in this way, to not mess with the `base` environment.